Frontend/src/Components/Shared/misc-constants.js

```diff
@@ -2,3 +2,10 @@
 export const MILLIS_PER_SEC = 1000;
 export const MILLIS_PER_MIN = 60000;
 export const MILLIS_PER_HR = 3600000;
+
+// FaultsView constant(s)
+export const IMAGE_HEIGHT = "7vh";
+
+// IOView constant(s)
+export const FS = "1.2em";
+export const FSS = "1.0em";
```

Comment:
I think these may be better left in their individual files since they're UI properties that are specific to those components. It'd be easier to find/make sense of and change them in those files. If FS and FSS were used in multiple files, I could see them being useful constants in a styling file. However, I think that would be better to save for if we decide to make global named/standard font sizes.